lib/screens/onboarding_screen/onboarding_screen.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:flutter/material.dart';
+
+
+
+
+class OnBordingScreen extends StatefulWidget {
+  const OnBordingScreen({super.key});
+
+  @override
+  State<OnBordingScreen> createState() => _OnBordingScreenState();
+}
+
+class _OnBordingScreenState extends State<OnBordingScreen> {
+       final PageController _pageController= PageController();
+
+       int _pageIndex = 0;
+
+    @override
+  void dispose() {
+    _pageController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body:SafeArea(
+        child: Column(
+          children: [
+            Expanded(
+              child: PageView.builder(
+                itemCount: demo_data.length,
+                controller: _pageController,
+
+                onPageChanged: (index){
+                  setState(() {
+                    _pageIndex = index;
+                  });
+                },
+
+
+                
+              itemBuilder: (context, index) =>Onbordingcontent(
+                image: demo_data[index].image,
+                title: demo_data[index].title,
+                description: demo_data[index].description,
+              ), 
+                  ),
+            ),
+
+            Row(
+              children: [
+               
+                 SizedBox(
+              height: 40,
+              width: 80,
+              child: ElevatedButton(
```

Comment:
Use a textbutton or i believe it's name is now flatbutton here. Remove the Sizedbox and give the button a padding so that text will not cause UI overflows